docs/INSTALL.md

```diff
@@ -100,6 +99,18 @@ sudo apt-get update && sudo apt-get install -y \
     zlib1g-dev
 ```
 
+For Ubuntu 20.04 the version of `cmake` that ships with Ubuntu is too old; a more recent version can be installed using `snap`:
+
+```bash
+sudo snap install --classic cmake
+```
+
+For Ubuntu 22.04 `cmake` may be installed using `snap` as well, or alternatively by using `apt`:
```

Comment:
Optionally, leave this out altogether and just point to `snap` for 22.04 as well.